src/microbe_stage/Microbe.Interior.cs

```diff
@@ -1323,16 +1332,58 @@ private void OnOrganelleRemoved(PlacedOrganelle organelle)
         cachedRotationSpeed = null;
         organelleMaxRenderPriorityDirty = true;
 
-        Compounds.Capacity = organellesCapacity;
+        UpdateCompoundBagCapacities();
     }
 
     /// <summary>
     ///   Recomputes storage from organelles, used after loading a save
     /// </summary>
     private void RecomputeOrganelleCapacity()
     {
-        organellesCapacity = organelles!.Sum(o => o.StorageCapacity);
-        Compounds.Capacity = organellesCapacity;
+        foreach (PlacedOrganelle organelle in organelles!)
+            UpdateCapacity(organelle, false);
+
+        UpdateCompoundBagCapacities();
+    }
+
+    /// <summary>
+    ///   Updates <see cref="organellesCapacity"/>
```

Comment:
I'm noticing that you are wrapping the documentation text really tightly? Is there a reason? The documentation comments should be filled all the way to column 120.